src/palak/FormAssignment1.java

```diff
@@ -0,0 +1,114 @@
+/*Selenium Assignment 1 : 16th Aug 2020
+1) Open the form in browser and verify Title is �TECHNOCREDITS�.
+2) Fill all details in the form, for Gender radio button if �Female� radio button is by default selected then 
+select �Male� radio button and vice versa.
+3) Reset the form, for that click on �Reset form in same tab� button.
+4) Verify Reset works properly on your text field, radio button and checkbox. (For each type, min one validation is required.)
+5) Click on the �Go and Practice for it� Button.
+6) Website redirect to �automationbykrishna.com�.
+7) Verify title and URL of redirected page website.
+Title should be �Login Signup Demo� and the URL should be �http://automationbykrishna.com/�.*/
+package palak;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+
+public class FormAssignment1 {
+
+	public static void main(String[] args) {
+		System.setProperty("webdriver.chrome.driver", "./resources/windows/chromedriver.exe");
+		WebDriver driver = new ChromeDriver();
+		driver.get("H:/TechnoCredits/First Form.html");
+
+		// verifying Title is �TECHNOCREDITS�.
+		String title = driver.getTitle();
+		if (title.equals("TECHNOCREDITS"))
+			System.out.println("Test Pass \n1.Title is ->" + title);
+		else
+			System.out.println("Test Fail Title is not " + title);
+
+		// Fill all the form
+		driver.findElement(By.id("first name")).sendKeys("Palash");
+		driver.findElement(By.id("last name")).sendKeys("Soni");
+		driver.findElement(By.id("E-mail")).sendKeys("Palash444@gmail.com");
+		driver.findElement(By.id("Company Name")).sendKeys("Vodafone");
+		
+		//Selecting Gender Value
+		WebElement genderValue = driver.findElement(By.name("gender"));
```

Comment:
using this name which value are you selecting, is not clear !!!

Reply:
Changes Done
`if(driver.findElement(By.id("femaleG")).isSelected());
		 	driver.findElement(By.id("maleG")).click();`